### DISCUSSION TITLE
Rooibos is associated with a hot beverage - what country is it native to?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#669] [Galaxy Trivia] [answered]
**Bonus:** How much caffeine does it have?

**Comment by gotthejuicee** [comment, marked as answer]
Rooibos is a type of herbal tea that is native to South Africa. It is made from the leaves of the Aspalathus linearis plant, which grows in the Western Cape region of South Africa. Rooibos has been a popular beverage in South Africa for centuries, and it is now enjoyed in many other countries around the world for its unique flavor and health benefits. Rooibos is naturally caffeine-free and is often used as a substitute for traditional tea or coffee. It can be served hot or cold and is often enjoyed with a little honey or milk.
